README.md

```diff
@@ -406,6 +406,8 @@ Visual Studio provides auto-generation of .edmx files, with the following proced
     -   In the solution explorer, expand the "Models" folder and delete the previous `<database name>_DB_MODELS.edmx` which represents the database to which you made changes; for example, if it was the CCT database that was changed, delete CCT_DB_MODELS.edmx by right clicking and selecting 'delete' (it's okay, we can remake it)
     -   At the bottom of the `web.config` file there is a `<connectionStrings>` tag with an `<add ...>` tag for every connection string nested within. Delete the connection string (the `<add ...>` tag) for the DB connection you are regenerating (if this is confusing, [this step](#connection-string) explains why you may need it deleted)
 
+    <!-- Do not know where this connection-string link above was supposed to go, so I have not changed it. -->
+
```

Comment:
If you look at line 426 of the raw README file, you can see there's a span tag with the id `connection-string`. This link is working correctly.

However, at some point in the past (I think it was probably my fault) the markdown indentation got messed up. It's definitely worthwhile to fix that so that it's actually readable.

Reply:
The markdown indentation doesn't seem off. What might be fixed about this?
![image](https://user-images.githubusercontent.com/60206522/109051741-d2506700-76a8-11eb-81d5-65547b3fea66.png)

Reply:
Oh, I got my spans confused. There's another one on line 426 and you'll see somehow the `-` got switched to `_` around that one which has messed up the bulleting and made it nearly unreadable.

Reply:
On 424? 426 looks like it might supposed to have been an underscore:
![image](https://user-images.githubusercontent.com/60206522/109052939-2f005180-76aa-11eb-9959-e931b42cca8d.png)

Reply:
All of the underscores on 426-431 are supposed to be dashes. 

![image](https://user-images.githubusercontent.com/35319956/109053956-4be95480-76ab-11eb-86cc-da597badfece.png)

Notice how the dashed version is actually bulleted and readable.

Reply:
![image](https://user-images.githubusercontent.com/60206522/109055449-3543fd00-76ad-11eb-9a3b-171c578b67bf.png)